plugins/org.eclipse.fordiac.ide.model/model/lib.ecore

```diff
@@ -1182,24 +1174,24 @@
         <details key="body" value="return org.eclipse.fordiac.ide.model.libraryElement.impl.FBNetworkElementAnnotations.getQualifiedName(this);"/>
       </eAnnotations>
     </eOperations>
-    <eOperations name="getWidth" eType="ecore:EDataType http://www.eclipse.org/emf/2003/XMLType#//Int">
+    <eOperations name="getWidth" eType="ecore:EDataType http://www.eclipse.org/emf/2003/XMLType#//Double">
       <eAnnotations source="http://www.eclipse.org/emf/2002/GenModel">
         <details key="body" value="throw new UnsupportedOperationException();"/>
       </eAnnotations>
     </eOperations>
-    <eOperations name="getHeight" eType="ecore:EDataType http://www.eclipse.org/emf/2003/XMLType#//Int">
+    <eOperations name="getHeight" eType="ecore:EDataType http://www.eclipse.org/emf/2003/XMLType#//Double">
       <eAnnotations source="http://www.eclipse.org/emf/2002/GenModel">
         <details key="body" value="throw new UnsupportedOperationException();"/>
       </eAnnotations>
     </eOperations>
     <eOperations name="getVisibleWidth" eType="ecore:EDataType http://www.eclipse.org/emf/2003/XMLType#//Int">
       <eAnnotations source="http://www.eclipse.org/emf/2002/GenModel">
-        <details key="body" value="return getWidth();"/>
+        <details key="body" value="return org.eclipse.fordiac.ide.model.CoordinateConverter.INSTANCE.iec61499ToScreen(getWidth());"/>
       </eAnnotations>
     </eOperations>
     <eOperations name="getVisibleHeight" eType="ecore:EDataType http://www.eclipse.org/emf/2003/XMLType#//Int">
       <eAnnotations source="http://www.eclipse.org/emf/2002/GenModel">
-        <details key="body" value="return getHeight();"/>
+        <details key="body" value="return org.eclipse.fordiac.ide.model.CoordinateConverter.INSTANCE.iec61499ToScreen(getHeight());"/>
       </eAnnotations>
     </eOperations>
```

Comment:
How about adding operations to get the *screen* height and width of an FB network element and then add derived attributes to cache the converted values in a group, subapp, or comment (reset on setHeight/setWidth)?

Reply:
That sounds interesting for an subsequent change. I first want to make sure that I haven't broken anything major.